python/pyarrow/table.pxi

```diff
@@ -5031,9 +5032,13 @@ def concat_tables(tables, c_bool promote=False, MemoryPool memory_pool=None):
     tables : iterable of pyarrow.Table objects
         Pyarrow tables to concatenate into a single Table.
     promote : bool, default False
-        If True, concatenate tables with null-filling and null type promotion.
+        If True, concatenate tables with null-filling and type promotion.
+        See field_merge_options for the type promotion behavior.
     memory_pool : MemoryPool, default None
         For memory allocations, if required, otherwise use default pool.
+    field_merge_options : FieldMergeOptions, default None
```

Comment:
Can we also allow passing "default" and "permissive" as strings?

Reply:
I like it, added